sdk/src/wallet/operations/transaction/build_transaction.rs

```diff
@@ -47,6 +48,15 @@ where
                 }
             }
 
+            // Inputs with timelock or expiration unlock condition require a CommitmentContextInput
+            if input
+                .output
+                .unlock_conditions()
+                .map_or(false, |u| u.iter().any(|u| u.is_timelock() || u.is_expiration()))
+            {
+                commitment_context_input_required = true;
```

Comment:
You could also set it to true in the if L44 so you can remove L90

Reply:
Hmm maybe not

Reply:
Had to add it, because there was no account